css/style.css

```diff
@@ -1,4 +1,5 @@
 @import url(/css/helper.css);
+@import url(/css/navbar.css);
 
 body {
     color: var(--bg-color-2);
```

Comment:
Might not be a good idea to directly style the body of your page
consider using a wrapper div and applying your styles to it